src/converter.ts

```diff
@@ -103,6 +112,19 @@ export class Converter {
     console.warn(...message);
   }
 
+  /**
+   * Log an rror message to console.error stream. Prefix the message string with `Error: `
+   * if it does not already have that text. Increments the `returnCode`.
+   * @param message parameters for console.warn
+   */
+  private error(...message) {
+    if (!message[0].startsWith('Error')) {
```

Comment:
Do we care about any errors beyond the first? This could also fail with an index out of bounds exception since we don't check the length first.

Reply:
this is varargs -- message[0] is the message template, so it must exist.